utils/python/CIME/XML/entry_id.py

```diff
@@ -176,3 +176,16 @@ def cleanupnode(self,node):
         if(vnode):
             node.remove(vnode)
         return node
+
```

Comment:
compare the value attribute in entry fields of different files with the same id attribute.